mslib/msui/mscolab.py

```diff
@@ -930,6 +937,49 @@ def handle_delete_operation(self):
             show_popup(self.ui, "Error", "Your Connection is expired. New Login required!")
             self.logout()
 
+    def set_operation_desc_label(self, op_desc):
+        self.active_operation_desc = op_desc
+        desc_count = len(str(self.active_operation_desc))
+        if desc_count < 95:
+            self.ui.activeOperationDesc.setText(
+                self.ui.tr(f"{self.active_operation_name}: {self.active_operation_desc}"))
+        else:
+            self.ui.activeOperationDesc.setText(
+                "Description is too long to show here, for long descriptions go "
+                "to operations menu.")
+
+    def update_description_handler(self):
+        # only after login
```

Comment:
comment hint because of the comment.

this must be protected on the server site (if not done)

The verify_user_token as client site function name is a bit misleading. It does lookup if not surpressed that the token of the user on the socket layer is still valiid. This token exists after login, but we can also disable this check. The token gets invalid when it expires on the server. e.g. the server gets rebootet. A liftetime is exceeded. ....